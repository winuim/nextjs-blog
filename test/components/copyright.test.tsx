import React from 'react';
import {render} from '../testUtils';
import Copyright from '../../components/copyright';

describe('Copyright component', () => {
  it('matches snapshot', () => {
    const {asFragment} = render(<Copyright />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
