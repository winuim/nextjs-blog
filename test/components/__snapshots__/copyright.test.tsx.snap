// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright component matches snapshot 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
  >
    Copyright Â© 
    <a
      class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
      href="https://material-ui.com/"
    >
      Your Website
    </a>
     2021.
  </p>
</DocumentFragment>
`;
